meta-luneos/recipes-support/sensorfw/sensorfw.bb

```diff
@@ -6,44 +6,33 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=2d5025d4aa3495befef8f17206a5b0a1"
 # We're potentially depending on libhybris so need to be MACHINE_ARCH
 PACKAGE_ARCH = "${MACHINE_ARCH}"
 
-PV = "0.11.4+git${SRCPV}"
-#PV = "0.12.6+git${SRCPV}"
-#SRCREV = "2e539015996111576a731750342effde7aaee87f"
-SRCREV = "6c1fa648fd934ef62fae64b304d106d61c64869f"
+PV = "0.14.4+git${SRCPV}"
+SRCREV = "b6e7f390962e5e5fa3677a393a6fb7ba46f2d67b"
 DEPENDS = "qtbase luna-sysmgr-common luna-service2 json-c glib-2.0 luna-sysmgr-ipc-messages"
 
-DEPENDS:append:halium = " libhybris virtual/android-headers libgbinder libglibutil "
+# Upstream Qt6 changes are pending merge in https://github.com/sailfishos/sensorfw/pull/16 
+# Once merged we can switch back to sailfishos mirror and drop the patch
+# git://github.com/sailfishos/sensorfw.git;protocol=https;branch=master
 
 SRC_URI = " \
-    git://github.com/webos-ports/sensorfw.git;protocol=https;branch=herrie/qt6 \
+    git://github.com/jmlich/sensorfw.git;protocol=https;branch=master \
 "
 
-# Note: maybe this should go in a bbappend in meta-pine64-luneos...
-SRC_URI:append:pinephone = " \
-    file://sensord-pinephone.conf \
-"
-SRC_URI:append:pinephonepro = " \
-    file://sensord-pinephonepro.conf \
-"
-
-SRC_URI:append:pinetab2 = " \
-    file://sensord-pinetab2.conf \
+# Upstream-Status: Backport
+SRC_URI:append = " \
+	file://0001-Add-trustlevel-for-enhanced-ACG.patch \
 "
 
-# Note: maybe this should go in a bbappend in meta-smartphone...
-SRC_URI:append:tenderloin = " \
-    file://sensord-tenderloin.conf \
-"
-
-SRC_URI:append:hammerhead = " \
-    file://sensord-hammerhead.conf \
+# Upstream-Status: Pending https://github.com/jmlich/sensorfw/pull/1
```

Comment:
This belongs to the .patch file and th value should be Submitted, read:
https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status